src/script/calling/CallingRepository.ts

```diff
@@ -98,6 +98,7 @@ export class CallingRepository {
   private readonly logger: Logger;
   private readonly callLog: string[];
   private readonly cbrEncoding: ko.Observable<number>;
+  private readonly useSftCalling: ko.Observable<boolean>;
```

Comment:
Please also add this to the constructor with a default value.

Reply:
Good catch. I was hoping for TypeScript to catch uninitialized paramters.